hook-tests/600-no-debian.test

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh -e
+#!/bin/sh
```

Comment:
Why is the `set -e` removed here?

Reply:
Well spotted Michael, that's added back in now with the latest commit.